compiler/rustc_borrowck/src/dataflow.rs

```diff
@@ -106,11 +106,11 @@ rustc_index::newtype_index! {
 /// `BorrowIndex`, and maps each such index to a `BorrowData`
 /// describing the borrow. These indexes are used for representing the
 /// borrows in compact bitvectors.
-pub struct Borrows<'a, 'mir, 'tcx> {
+pub struct Borrows<'mir, 'tcx> {
     tcx: TyCtxt<'tcx>,
     body: &'mir Body<'tcx>,
 
-    borrow_set: &'a BorrowSet<'tcx>,
+    borrow_set: &'mir BorrowSet<'tcx>,
```

Comment:
That lifetime seems wrong. The borrowset is not part of mir. We should just use `'a` to refer to short lived lifetimes

Reply:
Ok, I've switched to `'a` (and occasionally `'b`) everywhere instead of `'mir`.